src/components/Toolbar.js

```diff
@@ -56,7 +80,8 @@ export const Toolbar = connect(({ toolbarOverlay, settings: { dark } }) => ({
   }) => (
     <div>
       <div className={'toolbar-container'}>
-        <div className={'toolbar'}>
+        <div id={'toolbar'} className={'toolbar'}>
```

Comment:
You can simply do `id='toolbar'` when the property value is a constant string.

Reply:
Got it.